src/backend/access/appendonly/appendonly_blkdir_udf.c

```diff
@@ -40,7 +40,7 @@ Datum gp_aoblkdir(PG_FUNCTION_ARGS);
 Datum
 gp_aoblkdir(PG_FUNCTION_ARGS)
 {
-	Oid       	aoRelOid = PG_GETARG_OID(0);
+	Oid	aoRelOid;
```

Comment:
Why not revert and this change?

Reply:
It can not be initialized while squelching because function arguments are not accessible at that moment.